src/test_l1_dsVideoPort.c

```diff
@@ -1002,48 +959,39 @@ void test_l1_dsVideoPort_negative_dsGetSurroundMode(void) {
  * 
  */
 void test_l1_dsVideoPort_positive_dsIsVideoPortActive(void) {
-	gTestID = 15;
-	UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
-
-	dsError_t status;
-
-	intptr_t handle[NUM_OF_PORTS]={HANDLE_ARRAY_INIT};
-
-	bool isActiveArray1[NUM_OF_PORTS];
-	bool isActiveArray2[NUM_OF_PORTS];
-
-	// Step 01: Initialize video port system
-	status = dsVideoPortInit();
-	UT_ASSERT_EQUAL(status, dsERR_NONE);
-
-	// Step 02: Get the video port handle
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
-		status = dsGetVideoPort(kPorts[i].id.type, kPorts[i].id.index, &(handle[i]));
-		UT_ASSERT_EQUAL(status, dsERR_NONE);
-	}
-
-	// Step 03: Check whether each video port is active
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
-		status = dsIsVideoPortActive(handle[i], &isActiveArray1[i]);
-		UT_ASSERT_EQUAL(status, dsERR_NONE);
-	}
-
-	// Step 04: Repeat the active status check
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
-		status = dsIsVideoPortActive(handle[i], &isActiveArray2[i]);
-		UT_ASSERT_EQUAL(status, dsERR_NONE);
-	}
-
-	// Step 05: Compare the array values
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
-		UT_ASSERT_EQUAL(isActiveArray1[i], isActiveArray2[i]);
-	}
-
-	// Step 06: Terminate the video port system
-	status = dsVideoPortTerm();
-	UT_ASSERT_EQUAL(status, dsERR_NONE);
-
-	UT_LOG("\n Out %s\n", __FUNCTION__); 
+    gTestID = 15;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t status;
+
+    intptr_t handle[gDSvideoPort_NumberOfPorts];
+
+    bool isActiveArray1[gDSvideoPort_NumberOfPorts];
```

Comment:
in single loop only we are comparing so may not required array
other places also if not required remove array and use single variable